packages/oidc-middleware/src/connectUtil.js

```diff
@@ -29,13 +33,46 @@ connectUtil.createOIDCRouter = context => {
       path:callbackPath
     }
   } = context.options.routes;
-  oidcRouter.use(loginPath, connectUtil.createLoginHandler(context));
+  oidcRouter.use(loginPath, bodyParser.urlencoded({ extended: false}), connectUtil.createLoginHandler(context));
   oidcRouter.use(callbackPath, connectUtil.createCallbackHandler(context));
   return oidcRouter;
 };
 
 connectUtil.createLoginHandler = context => {
-  return passport.authenticate('oidc');
+  const passportHandler = passport.authenticate('oidc');
+  const csrfProtection = csrf();
```

Comment:
is this going to collide on the session key?

Reply:
Per discussion: there is a common dependency on the session storage location, which may create edge cases if they are trying to do CSRF validation on post requests before our middleware is invoked.  We will handle this sometime later.